apps/tangle-dapp/data/payouts/usePayoutsAvailability.ts

```diff
@@ -1,18 +1,16 @@
 import { useEffect, useState } from 'react';
 
-import useSubstrateAddress from '../../hooks/useSubstrateAddress';
 import usePayouts from '../NominationsPayouts/usePayouts';
 
 const usePayoutsAvailability = () => {
-  const activeSubstrateAddress = useSubstrateAddress();
-  const { data: payouts } = usePayouts(activeSubstrateAddress ?? '');
+  const payoutsData = usePayouts();
```

Comment:
Same here, pass the payout result into this hook, instead of re-call the `usePayouts()`.